idea_town/frontend/static-src/app/views/feedback-view.js

```diff
@@ -51,7 +51,6 @@ export default BaseView.extend({
   },
 
   cancel(e) {
-    this.animateRemove();
```

Comment:
@meandavejustice `this.animateRemove` just calls `this.remove` by default, so it seems like removing this line altogether will not remove the feedback view when the user dismisses it. Is that right, or am I missing something?